src/upgrade/nc_upgrade_scripts/1.0.0/config_dir_restructure.js

```diff
@@ -289,3 +293,11 @@ module.exports = {
     run,
     description: 'Config directory resturcture'
 };
+
+module.exports.move_old_accounts_dir = move_old_accounts_dir;
+module.exports.create_identity_if_missing = create_identity_if_missing;
```

Comment:
Could you add a comment with an example in the function `create_identity_if_missing` what is the difference between the `identity_path` and `identity_path` so it will be less confusing?

Reply:
do you mean identity_path and identity_dir_path?